src/core/sys/linux/perf_event.d

```diff
@@ -0,0 +1,2259 @@
+/**
+  * D header file for perf_event_open system call.
+  *
+  * Converted from linux userspace header, comments included.
+  *
+  * Authors: Max Haughton
+  */
+module core.sys.linux.perf_event;
+version (linux)  : extern (C):
+@nogc:
+nothrow:
+@system:
+
+import core.sys.posix.sys.ioctl;
+import core.sys.posix.unistd;
+
+version (X86_64)
+{
+        enum __NR_perf_event_open = 298;
+}
+else version (X86)
+{
+        enum __NR_perf_event_open = 336;
+}
+else version (ARM)
+{
+        enum __NR_perf_event_open = 364;
+}
+else version (ARM64)
```

Comment:
```suggestion
else version (AArch64)
```

The [official naming](https://dlang.org/spec/version.html) is not symmetric with `version (ARM)`.

Reply:
Good spot. The [kernel](https://www.kernel.org/doc/html/latest/arm64/index.html) refers to it as ARM64.